sparkle/src/components/ConversationMessageFeedbackSelector.tsx

```diff
@@ -58,21 +70,31 @@ export function FeedbackSelector(
       <PopoverRoot open={isPopoverOpen}>
         <PopoverTrigger asChild>
           <div className="s-flex s-items-center">
-            <Button
-              variant={feedback?.thumb === "up" ? "highlight" : "outline"}
-              size="xs"
-              disabled={isSubmittingThumb}
-              onClick={() => selectThumb("up")}
-              className={"s-rounded-r-none s-border-r-0"}
-              icon={HandThumbUpIcon}
+            <Tooltip
+              label="I found this helpful"
```

Comment:
I think you're gonna need `tooltipTriggerAsChild` otherwise your console is gonna shout

Reply:
For reference: https://www.radix-ui.com/primitives/docs/guides/composition